public/static/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -0,0 +1,115 @@
+[OAuth](https://developers.google.com/identity/protocols/OAuth2) credentials
+(**client ID** and **client secret**).
+
+1. Sign into the [Google API Console](https://console.developers.google.com).
+
+   > Double check you're using the intended Google account (upper-right corner).
+
+2. Select or
+   [Create](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project)
+   a project for DVC remote connections.
+
+3. [Enable the Drive API](https://developers.google.com/drive/api/v2/about-sdk)
+   from the **APIs & Services** **Dashboard** (left sidebar), click on **+
+   ENABLE APIS AND SERVICES**. Find and select the "Google Drive API" in the API
+   Library, and click on the **ENABLE** button.
+
+   ![](/static/img/gdrive-enable-apis-and-services.png)
+
+4. Go back to **APIs & Services** in the left sidebar, and select **OAuth
+   consent screen**. Chose a **User Type** and click **CREATE**. On the next
+   screen, enter an **Application name** e.g. "DVC remote storage", and click
+   the **Save** (scroll to bottom).
+
+5. From the left sidebar, select **Credentials**, and click the **Create
+   credentials** dropdown to select **OAuth client ID**. Chose **Other** and
+   click **Create** to proceed with a default client name.
+
+   ![](/static/img/gdrive-create-credentials.png)
+
+6. The newly generated **client ID** and **client secret** should be shown to
+   you now, and you can always come back to **Credentials** to fetch them.
+
+⚠️ It should be safe to share **client ID** and **client secret** among your
+team. These credentials are only used to generate the
+[authorization](#authorization) DVC will later need in order to connect to the
+Google Drive.
+
+> Note that Google Drive API has usage limits/quotas per credentials in a
+> _project_ (which you can review in the
+> [OAuth consent screen](https://console.developers.google.com/apis/credentials/consent)).
+> Please keep this in mind when sharing them, or you may
+> [exceed the limits](https://developers.google.com/drive/api/v2/handle-errors?hl=ro#resolve_a_403_error_usage_limit_exceeded).
+
+## URL format for Google Drive remotes
+
+A GDrive DVC remote URLs is needed for the `dvc remote add` command. It can be
+constructed with a _base_, and an **optional** _path_ to a folder i.e.
+`gdrive://<base>/path/to/folder`. The base can be one of:
+
+1. `root` - indicates your topmost Google Drive directory.
+
+  ```dvc
+  $ dvc remote add -d mygdroot gdrive://root
+  $ dvc remote add mygdfolder gdrive://root/path/to/folder
+  ```
+
+2. Google Drive _Folder ID_
+
+  To obtain the unique _Folder ID_ in question, navigate into that folder in
+  your web browser, and find it in the address bar. For example, for
+  `https://drive.google.com/drive/folders/0AIac4JZqHhKmUk9PDA`:
+
+  ```dvc
+  $ dvc remote add mygdfolder gdrive://0AIac4JZqHhKmUk9PDA
+  $ dvc remote add mygdsubfolder \
+                   gdrive://0AIac4JZqHhKmUk9PDA/sub/path
+  ```
+
+  > Note that
```

Comment:
can make it regular text and probably even make it the main sentence on this section. Probably, not that many reasons to access a regular folder by id, but for the team/shared drive it's the only option as far as I know.